### PR TITLE
Add scale ratio to button touchdown

### PR DESCRIPTION
**Pull request by v1ctor**
I added ability to specify scale ratio in ControlButton, while the button is highlighted.

### CODE REVIEW

File: extensions/GUI/CCControlExtension/CCControlButton.cpp [hunk 1]
```diff
@@ -98,8 +99,6 @@ bool ControlButton::initWithLabelAndBackgroundSprite(Node* node, Scale9Sprite* b
 
         setTouchEnabled(true);
         _isPushed = false;
-        _zoomOnTouchDown = true;
```
Comment (dumganhar) [inline extensions/GUI/CCControlExtension/CCControlButton.cpp:100]: Why did you delete this line ?

**Comment by dumganhar** [comment]
Please pull the latest codes of develop branch to resolve travis ci issue! Thanks.

File: extensions/GUI/CCControlExtension/CCControlButton.cpp (hunk 1 above)
Comment (v1ctor) [inline extensions/GUI/CCControlExtension/CCControlButton.cpp:100]: It is duplicated below in the line 108.

**Comment by dumganhar** [comment]
Merged. Thanks.
